src/reducers/index.js

```diff
@@ -289,7 +289,10 @@ export function getSecrets(
   state,
   { namespace = getSelectedNamespace(state) } = {}
 ) {
-  return secretSelectors.getSecrets(state.secrets, namespace);
+  const secretsList = secretSelectors.getSecrets(state.secrets, namespace);
+  return secretsList.filter(secret => {
```

Comment:
rather than filter on the client we should request only the secrets of interest

we can filter by type on the API, just like we did on the back end using the fieldSelector, e.g.:

http://localhost:9097/proxy/api/v1/namespaces/default/secrets?fieldSelector=type=kubernetes.io/basic-auth

Reply:
Added your code in, thank you